src/silx/test/utils.py

```diff
@@ -118,12 +118,12 @@ def configure(self, parsed_options=None):
                 self.WITH_GL_TEST = False
                 self.WITH_GL_TEST_REASON = "OpenGL package not available"
 
-        if parsed_options is not None and parsed_options.low_mem:
-            self.TEST_LOW_MEM = True
-            self.TEST_LOW_MEM_REASON = "Skipped by command line"
-        elif os.environ.get("SILX_TEST_LOW_MEM", "True") == "False":
-            self.TEST_LOW_MEM = True
-            self.TEST_LOW_MEM_REASON = "Skipped by SILX_TEST_LOW_MEM env var"
+        if parsed_options is not None and not parsed_options.high_mem:
+            self.WITH_HIGH_MEM_TEST = False
+            self.WITH_HIGH_MEM_TEST_REASON = "Skipped by command line"
+        elif os.environ.get("WITH_HIGH_MEM_TEST") == "False":
+            self.WITH_HIGH_MEM_TEST = False
+            self.WITH_HIGH_MEM_TEST_REASON = "Skipped by WITH_HIGH_MEM_TEST env var"
```

Comment:
I may miss something here, but it looks you can never set `self.WITH_HIGH_MEM_TEST` to `True`.